src/docs/sphinx/quickstart.rst

```diff
@@ -177,40 +177,72 @@ container. Instructions for this process are located :ref:`here <docker-label>`.
 Building Serac
 --------------
 
-Serac uses a CMake build system that wraps its configure step with a script
-called ``config-build.py``.  This script creates a build directory and
-runs the necessary CMake command for you. You just need to point the script
-at the generated or a provided host-config. This can be accomplished with
-one of the following commands:
+Serac is a CMake project. Configure with at least the ``-C``
+flag below, to ensure CMake can locate the dependencies installed by spack:
 
 .. code-block:: bash
 
-   # If you built Serac's dependencies yourself either via Spack or by hand
-   $ python3 ./config-build.py -hc <config_dependent_name>.cmake
+  $ cmake . -B build \
```

Comment:
In the past, having the source directory, '.' in this case, before the initial cache, '-C', caused a lot of very bad behavior in CMake. I think what happened is it delays loading the initial cache. I can't seem to find any articles to explain this. I am not sure this is still the case, but I would not trust CMake to do the correct thing. 

Please, change this line to have the `.` at the very end of the command line.